ale_linters/javascript/jscs.vim

```diff
@@ -1,9 +1,63 @@
 " Author: Chris Kyrouac - https://github.com/fijshion
 " Description: jscs for JavaScript files
 
+call ale#Set('javascript_jscs_executable', 'jscs')
+call ale#Set('javascript_jscs_use_global', 0)
```

Comment:
Add some documentation entries for these variables. They just need to link back to the explanation of how the `node_modules` loading works like `eslint`.